mobile-first/css/styles.css

```diff
@@ -0,0 +1,449 @@
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+		font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+p {
+  line-height: 18px;
+}
+
+body {
+	margin: 0 auto;
+	max-width: 500px;
+	display: block;
+	position: relative;
+}
+/* navbar */
+
+.container nav {
+	position: fixed;
+	top: 0;
+	background-color: white;
+	z-index: 500;
+	border-bottom: 1px solid grey;
+  width: 500px;
+  border-bottom: 1px solid black;
+  margin: 0;
+  padding: 0 0 0 0;
+}
+
+.container .burger {
+  display: block;
+  position: absolute;
+  height: 60px;
+  top: 0;
+  right: 0;
+  margin-top: 10px;
+  cursor: pointer;
+}
+
+.container .navcontainer .logo {
+  margin: 0 0 18px 0;
+  padding: 0 0 20px 0;
+}
+
+.container nav {
+  padding-top: 20px;
+}
+
+.container .navcontainer {
+  display: flex;
+  flex-direction: column-reverse;
+  text-align: center;
+}
+
+.container .navcontainer .navmenu {
+  justify-content: center;
+  flex-direction: column;
+  width: 500px;
+  display: flex;
+}
+
+.container .navcontainer .navmenu a{
+  padding: 20px 0 20px 0;
+  text-align: center;
+  border-top: 1px solid black;
+  font-size: 1.3em;
+}
+
+#menu-links {
+  display: none;
+}
+
+.last-menu {
+  border-bottom: 1px solid black;
+}
+
+.navmenu a {
+	
+	text-decoration: none;
+	color: grey;
+	
+}
+
+.navmenu a:hover {
+	text-decoration: underline;
+
+}
+
+/* header */
+
+.container .headermain img {
+  display: none;
+}
+
+
+.headermain {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	padding: 80px 0 32px 0;
+	margin: 80px 0 43.2px 0;
+}
+
+.headertext {
+	flex: 0;
+	text-align: center;
+	display: flex;
+	flex-direction: column;
+  align-items: center;
+}
+
+.headertext h1 {
+	font-size: 72px;
+}
+
+.home-page .headermain .headertext .get-started {
+	border: 1px solid black;
+	padding: 6.4px 16px;
+	width: 169.6px;
+	cursor: pointer;
+	text-align: center;
+	font-size: 1.1rem;
+	margin-top: 6.4px;
+}
+
+.home-page .headermain .headertext .get-started:hover {
+	background-color: black;
+	color: white;
+}
+
+.headermain .logo {
+	flex: 0;
+}
+
+/* above image content */
+
+.container .above-img {
+  display: flex;
+  flex-direction: column;
+  padding: 52px 20px 0 20px;
+}
+
+.container .above-img .featuress {
+  padding-bottom: 46px;
+}
+
+.container .above-img .about {
+  padding-bottom: 53px;
+}
+
+/* middle image  */
+
+.container .middle-img {
+  padding: 0 20px 30px 20px;
+  width: 100%;
+}
+
+/* below image content */
+
+.container .below-img {
+  padding: 0 20px;
+}
+
+.container .below-img .product {
+    padding: 63px 0 0 0;
+}
+
+.container .below-img .vision {
+  padding: 64px 0 30px 0;
+}
+
+.container address {
+  padding: 25px 20px 0 20px;
+}
+
+footer {
+  padding-top: 25px;
+  text-align: center;
+}
+
+@media (min-width: 800px) {
+  body {
+    margin: 0 auto;
+    max-width: 800px;
```

Comment:
This is where you'd want to set width to 100% instead of setting another min/max width.